lib/board.rb

```diff
@@ -28,47 +28,53 @@ def valid_coordinate?(coordinate)
   end
 
   def valid_placement?(ship, coordinate_array)
+    return false if lengths_differ(ship, coordinate_array)
+    return false if any_coordinates_invalid(coordinate_array)
+    return false if any_coordinates_already_hold_ship(coordinate_array)
+    return false if coordinates_not_consecutive(ship, coordinate_array)
+    # All validations passed, therefore it is a valid placement
+    return true
+  end
 
-    # Test if correct Length
-    return false if ship.length != coordinate_array.length
+  def lengths_differ(ship, coordinate_array)
+    ship.length != coordinate_array.length
```

Comment:
Is this consistent with our adding / not adding `return`s to methods?

I thought we were adding returns -- just wanted to check

Reply:
Same with other method